### PR TITLE
Adds more map interactions and differentiation of active state in desktop

### PR DESCRIPTION
**Pull request by jlhogan**

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -24,7 +24,7 @@ body {
   color: white;
 }
 
-.usa-button:not(.usa-button--outline):not(.usa-button--base) {
+.usa-button:not(.usa-button--outline):not(.usa-button--base):not(.usa-button--outline):not(.usa-button--unstyled) {
```
Comment (cathydeng) [inline src/index.css:27]: `:not(.usa-button--outline)` twice?

File: src/components/Search/ResultsList.tsx [hunk 2]
```diff
@@ -3,20 +3,36 @@ import ResultCard from "./ResultCard";
...
+  "border-bottom border-base-lighter padding-y-3 padding-x-2";
+const DESKTOP_CLASSES_ACTIVE =
+  "border border-width-2px radius-sm border-primary-light padding-y-3 padding-x-2";
 const MOBILE_CLASSES =
   "border border-base-lighter radius-lg padding-2 margin-bottom-1";
 
-function ResultsList({ results, isMobile = false }: ResultsListProps) {
+function ResultsList({
+  results,
+  activeId,
```
Comment (cathydeng) [inline src/components/Search/ResultsList.tsx:19]: nit: `activeResultId` or `selectedResultId` would make this more immediately clear

**Review by cathydeng — APPROVED** [review]
functionality LGTM! two pieces of design feedback:
- more urgent: on mobile, the selected result card is below the fold, so it may be easy for the user to miss the ui response when they click the map. maybe we can address this w/ some combo of making the content above the map take up less real estate + making the map shorter? for reference, i think eater makes a [nice map experience on mobile](https://sf.eater.com/maps/best-coffee-shops-san-francisco) - default to list view, and map view is a separate pane without any of the high-level context about the list (in our case zip + result count + filters), but this may also be too much of a ui rework for soft launch. cc @admoorgit for thoughts
- the blue outline looks a lil funky on desktop b/c the right side of the box is directly next to the map. maybe a background color on the box would look nicer?

File: src/index.css (hunk 1 above)
Comment (jlhogan) [reply src/index.css:27]: whoops

**Review by cathydeng — APPROVED** [review]
🗺️

File: src/index.css (hunk 1 above)
Comment (admoorgit) [reply src/index.css]: Hello! A few possible fixes to move things up:
- Snip the space above the "Results" line ... remove the .67em margin above
- Move the directions (Tap a flag...) underneath the map where the card would show up
- Put List / Map / Filters on the same row, like:  (  Filter  )  (   List   |   Map   )

Comment (admoorgit) [reply src/index.css]: Example: 
image

**Pull request merged**